server/server/views/profile.py

```diff
@@ -0,0 +1,49 @@
+import json
+import transaction
+
+from pyramid.config import Configurator
+from pyramid.response import Response
+from pyramid.view import view_config
+
+from server.models import get_dbsession
+
+from server.models import Base
+from server.models.user import User
+from server.models.user import users_to_json
+from server.models.user import datetime_conv, AlchemyEncoder
+
+@view_config(route_name='users.view', renderer='json')
+def get_users(request):
+    qry = get_dbsession().query(User).all()
+    return json.loads(json.dumps(qry, cls=AlchemyEncoder))
+
+@view_config(route_name='users.create', renderer='json')
+def create_user(request):
+    json_str = request.json_body
+    json_encoded = json.dumps(json_str)
+    json_decoded = json.loads(json_encoded)
```

Comment:
It's unclear for me what is a purpose of this actions? You takes decoded JSON from body, encodes it JSON and then decodes back. It's a bit weird.